### PR TITLE
Package dependency

### PR DESCRIPTION
**Pull request by pronav**
Added script to update node's resolution paths

### CODE REVIEW

File: bin/serve [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  if (Array.isArray(argv.proxy)) {
+    proxies = argv.proxy;
+  } else if (argv.proxy) {
+    proxies = [argv.proxy];
+  } else {
+    proxies = [];
+  }
+  return proxies;
+}
+
```
Comment (sidx1024) [inline bin/serve]:
```suggestion
const getProxies() => {
  return [].concat(argv.proxy).filter(Boolean);
}
```

Comment (umanghome) [reply bin/serve]: @sidx1024 Why do you need the filter?

Comment (sidx1024) [reply bin/serve]: If `argv.proxy` is null then it returns an empty array, not an array with null in it.

File: bin/serve [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const getHandler = () => {
+  const proxies = getProxies().map(p => {
+    const split = p.split('=');
+    return [split[0], split[1]];
+  })
+  return (req, res) => {
+    const proxyTarget = proxies.find(p => req.url.startsWith(p[0]));
```
Comment (umanghome) [inline bin/serve]: Problem with this: `http://localhost:8000/apisarecool/test/layout.php` makes a request to `https://api.razorpay.comsarecool/test/layout.php`.

**Pull request merged**